test/test_browser.py

```diff
@@ -4622,6 +4625,8 @@ def test_webgl_from_client_side_memory_without_default_enabled_extensions(self):
   # For testing WebGL draft extensions like this, if using chrome as the browser,
   # We might want to append the --enable-webgl-draft-extensions to the EMTEST_BROWSER env arg.
   @requires_graphics_hardware
+  @no_2gb('multiDrawElementsInstancedWEBGL: size exceeds the supported range')
+  @no_4gb('multiDrawElementsInstancedWEBGL: size exceeds the supported range')
```

Comment:
just out of curiosity, is this a browser limitation?

Reply:
It seems to be yes. @kainino0x I this a known limitation?   Should I open a chromium bug?

Reply:
I think we just missed this when we updated all the other WebGL entry points. Filed https://crbug.com/324562920